### DISCUSSION TITLE
JavaScript vs Typescript

### DISCUSSION THREAD

**Discussion by nirajgiriXD** [nirajgiriXD/garden-of-quotes#14] [Q&A] [answered]
For projects like [Garden of Quotes](https://github.com/nirajgiriXD/garden-of-quotes/), which one is better: `JavaScript` or `TypeScript`?

**Comment by sumanjha108155** [comment, marked as answer]
For smaller, simpler projects or quicker development cycles, JavaScript may be preferable due to its lower learning curve and dynamic nature. TypeScript, being a superset of JavaScript, is advantageous for larger projects, offering static typing for early error detection, enhanced code maintainability, and better tooling support. TypeScript's type system aids team collaboration and documentation and for project like this TypeScript seems to be a better choice.
